ops/docker-compose.yml

```diff
@@ -140,23 +140,67 @@ services:
     build:
       context: ..
       dockerfile: ./ops/docker/Dockerfile.geth
-    # override with the geth script and the env vars required for it
     entrypoint: sh ./geth.sh
     env_file:
       - ./envs/geth.env
     environment:
         ETH1_HTTP: http://l1_chain:8545
         ROLLUP_STATE_DUMP_PATH: http://deployer:8081/state-dump.latest.json
-        # used for getting the addresses
         URL: http://deployer:8081/addresses.json
-        # connecting to the DTL
         ROLLUP_CLIENT_HTTP: http://dtl:7878
+        ROLLUP_BACKEND: 'l1'
         ETH1_CTC_DEPLOYMENT_HEIGHT: 8
         RETRIES: 60
         IS_VERIFIER: "true"
     ports:
       - ${VERIFIER_HTTP_PORT:-8547}:8545
       - ${VERIFIER_WS_PORT:-8548}:8546
+  
+  l2_dtl:
+    depends_on:
+      - l2geth
+    image: ethereumoptimism/data-transport-layer
+    deploy:
+      replicas: 0
+    build:
+      context: ..
+      dockerfile: ./ops/docker/Dockerfile.data-transport-layer
+    entrypoint: ./dtl.sh
+    env_file:
+      -  ./envs/dtl.env
+    environment:
+        URL: http://deployer:8081/addresses.json
+        DATA_TRANSPORT_LAYER__SYNC_FROM_L1: 'false'
+        DATA_TRANSPORT_LAYER__SYNC_FROM_L2: 'true'
+        DATA_TRANSPORT_LAYER_DEFAULT_BACKEND: 'l2'
+        DATA_TRANSPORT_LAYER__L2_RPC_ENDPOINT: http://l2geth:8545
+        DATA_TRANSPORT_LAYER__L2_CHAIN_ID: 420
+    ports:
+      - ${L2_DTL_PORT:-7879}:7878
```

Comment:
Do we need another DTL? I thought that @tynes made it so that l2geth nodes can query if they want to use L1 or L2 as the DTL backend.

Reply:
talked to @tynes and will switch this to the singular dtl

Reply:
^just tried using the existing DTL and crashed the whole stack when i started the integration test due to [this error](https://github.com/ethereum-optimism/optimism/pull/981/files#r641994439) in the comment above; then when i set the highestBlock to default to 0 i get this error instead

```
Cannot read property 'index' of undefined","stack":"TypeError: Cannot read property 'index' of undefined\n    at Object.parseBlock (/opt/optimism/packages/data-transport-layer/dist/src/services/l2-ingestion/handlers/transaction.js:10:46)\n    at L2IngestionService._syncSequencerBlocks (/opt/optimism/packages/data-transport-layer/dist/src/services/l2-ingestion/service.js:121:68)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async L2IngestionService._start (/opt/optimism/packages/data-transport-layer/dist/src/services/l2-ingestion/service.js:69:17)\n    at async L2IngestionService.start (/opt/optimism/packages/common-ts/dist/base-service.js:30:9)\n    at async Promise.all (index 2)\n    at async L1DataTransportService._start (/opt/optimism/packages/data-transport-layer/dist/src/services/main/service.js:48:9)\n    at async L1DataTransportService.start (/opt/optimism/packages/common-ts/dist/base-service.js:30:9)\n    at async /opt/optimism/packages/data-transport-layer/dist/src/services/run.js:61:9"},"msg":"Caught an unhandled error"
```
think the cause is we can't read the genesis block correctly in the dtl, which i could try fix in a followup PR but isn't within scope of this one. will keep this setup for now and chat with the team to see if there's a cleaner solution

Reply:
The DTL should never try to ingest the genesis block as it has no transactions

Reply:
This may be useful for deduplication of config: https://nickjanetakis.com/blog/docker-tip-82-using-yaml-anchors-and-x-properties-in-docker-compose